src/main/java/com/Leapwork/Leapwork_plugin/PluginHandler.java

```diff
@@ -602,7 +616,7 @@ public String getRunStatus(AsyncHttpClient client, String controllerApiHttpAddre
 
         }
     }
-
+    @SuppressFBWarnings(value = "DLS_DEAD_LOCAL_STORE")
```

Comment:
What is it and why it is needed?

Reply:
this was required for suppressing given by spotbugs in maven,